neatnet/artifacts.py

```diff
@@ -68,16 +68,20 @@ class FaceArtifacts:
 
     Examples
     --------
-    >>> fa = neatnet.FaceArtifacts(street_network_prague)
+    >>> import neatnet
+    >>> street_network = neatnet.utils.fetch_milton_keynes()
```

Comment:
Can you print head of `street_network` after this line before using neatnet? It is unclear what this is. A graph? A gdf? Path to a file? Something else?